README.md

```diff
@@ -1,2 +1,4 @@
 # node-ari-shared-line-appearances
 Co-op implementation of SLA in ARI using Node.js
+
+This implementation assumes that you have at least Asterisk 12.0.0 running. This is the version where ARI first came around, and it should have all of the functions required for this project. You must also have a valid ARI user in ari.conf named user and have a password "pass", as well as have 127.0.0.1 (or localhost) and 8088 configured as the bindaddr and bindport respectively in http.conf. You must have a SIP user named phone, or change the originate to match a valid SIP device.
```

Comment:
Things like the ARI user/pass and the SIP endpoint (or other technologies such as PJSIP) should not be given here. These values should come from a configuration file.